### DISCUSSION TITLE
Looking for an iTwin Synchronization API sample that uses ProjectWise documents as the source.

### DISCUSSION THREAD

**Discussion by JohnMajerle** [iTwin/itwinjs-core#5106] [Q&A]
I have recently viewed learning path module:

[iTwin Developer Accreditation The Synchronization API](https://education.bentley.com/LearningPaths/guided-learningpaths-804)

Are there any samples available where the source dgn files are managed in ProjectWise?  My customer stores all their dgn files in ProjectWise.

**Comment by candcconsulting** [comment]
Hi @JohnMajerle did you see this wiki article
https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/64014/how-to-build-your-own-synchronizer
Also note that the connectors have the capability to publish directly from PW ...
